department-of-veterans-affairs/abd-vro
# #2582 Adding a datadog helm template for datadog api key

This template is now available for any other containers that need to use the datadog rest api

## What was the problem?

We need the ep-merge app container to receive a secret as an environment variable. Other containers will want to use the same secret in the future, so a global shared template makes the most sense.

Associated tickets or Slack threads:
- #2582

## How does this fix it?[^1]
Adds the global shared template _datadog.tpl and then propagates that into the ep-merge deployment.

## How to test this PR
We just need to validate the values all come out correctly post helm deploy to dev.


[^1]: [Pull-Requests guidelines](https://github.com/department-of-veterans-affairs/abd-vro/wiki/Pull-Requests). If PR is significant, update [Current Software State](https://github.com/department-of-veterans-affairs/abd-vro/wiki/Current-Software-State) wiki page.
[^secrel]: To check if a PR will succeed in the SecRel workflow, [test PRs in the SecRel pipeline](https://github.com/department-of-veterans-affairs/abd-vro-internal/wiki/Secure-Release-process#to-test-prs-in-the-secrel-pipeline).
